app/models/creator.rb

```diff
@@ -0,0 +1,115 @@
+class Creator < ActiveRecord::Base
+  has_many :content
+  has_and_belongs_to_many :users
+  validates :p_id, :provider, :username, presence: true
+
+  include HTTParty
+  LIMIT_PER_PAGE = 10
+
+  #method to configure twitter
+  def twit
+    Seemore::Application.config.twitter
+  end
+
+  def has_image?
+    if avatar_url.nil?
+      false
+    else
+      true
+    end
```

Comment:
Or even better, you don't need a conditional at all. `.any?` will return true and false for you, so this could be condensed to simply:
```
def has_image?
   avatar_url.any?
end
```